streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1013,12 +1017,16 @@ private static Metrics getMetrics(final StreamsConfig config, final Time time, f
                 final int threadIdx = getNextThreadIndex();
                 final int numLiveThreads = getNumLiveStreamThreads();
                 final long cacheSizePerThread = getCacheSizePerThread(numLiveThreads + 1);
+                final long maxBufferSizePerThread = getBufferSizePerThread(numLiveThreads + 1);
                 log.info("Adding StreamThread-{}, there will now be {} live threads and the new cache size per thread is {}",
                          threadIdx, numLiveThreads + 1, cacheSizePerThread);
                 resizeThreadCache(cacheSizePerThread);
+                log.info("Adding StreamThread-{}, there will now be {} live threads and the new buffer size per thread is {}",
```

Comment:
Seems kind of weird to log this twice, it makes it seem like we're adding this StreamThread a second time. How about instead just adding the new buffer size to the original log message (eg something like)
```suggestion
                log.info("Adding StreamThread-{}, there are now {} threads with a buffer size {} and cache size {} per thread.",
```